components/LobbyCircle.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styles from '../styles/home.module.scss';
```

Comment:
this is a component so it should have its own `scss` file! call it `LobbyCircle.module.scss`